src/fpm_compiler.F90

```diff
@@ -173,6 +173,7 @@ module fpm_compiler
     flag_intel_warn = " -warn all", &
     flag_intel_check = " -check all", &
     flag_intel_debug = " -O0 -g", &
+    flag_intel_opt = " -Ofast", &
```

Comment:
I had tried to avoid that because `-fast` includes `-xHost`, which introduces instructions specific to the current CPU (which may not run on others), while `-Ofast` does not: [https://www.intel.com/content/www/us/en/docs/fortran-compiler/developer-guide-reference/2023-0/alphabetical-option-list.html](intel compiler options)

Reply:
I just saw that [Ofast only supports macOS and Linux, not Windows](https://www.intel.com/content/www/us/en/docs/fortran-compiler/developer-guide-reference/2023-0/ofast.html), so it is true that `-O3` would be more appropriate.

Reply:
Agree: makes sense to just use the `O3` option.